general/development/tools/component-library.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Component library
+tags:
+  - Coding_guidelines
+  - Core_development
+  - Developer_tools
+  - Guidelines_for_contributors
+  - Interfaces
+  - Moodle_User_Interface_Guidelines
+  - Processes
+  - Themes
+  - Usability
+---
+
+## Component library
+
+This document provides a brief overview of the Moodle Component Library.
+
+## What is the Component Library?
+
```

Comment:
I'd recommend just removing these lines. 
The document already has an h1 title of 'Component Library' so adding an h2 with the same thing is superflous and not great for accessibility.

The 'What is the Component LIbrary?' title is really just an introduction to the content and that's where we should start.

```suggestion
```

Reply:
I have removed these lines as per the review in the updated pr.